www/src/pages/editor/DedupMineralSiteTable.tsx

```diff
@@ -292,6 +292,7 @@ export const DedupMineralSiteTable: React.FC<DedupMineralSiteTableProps> = obser
           expandedRowKeys: [...(editingDedupSite ? [editingDedupSite] : [])],
         }}
       />
+      <NewMineralSiteModal commodity={commodity!} visible={isModalVisible} onClose={() => setIsModalVisible(false)} />
```

Comment:
rename isModalVIsible

Reply:
I think we should move NewMineralSiteModal to outside of DedupMineralSiteTable. Put it in same level as the Search component.